backend/utils/seed.ts

```diff
@@ -0,0 +1,148 @@
+import { faker } from '@faker-js/faker'
+import prisma from './prismaClient';
+
+const db = prisma
+
+const userData = Array.from({ length: 20 }).map(() => ({
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    email: faker.internet.email()
+}))
+
+
+const main = async () => {
+
+    //create many users
+    await db.user.createMany({
+        data: userData
```

Comment:
given this is a small script, performance isn't super important but you could just do all of these calls at once


```
await db.user.createMany({
    data: [...userdata, {
        /* hyun user */ 
    }, { 
        /* john user */
    }, {
        /* sarah user */
    }]
})

```

Reply:
I created the variable for each users (hyun, sarah, and john) so I can conveniently access their properties for other queries.